.github/workflows/ci.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Setup Python (for Cask)
         uses: actions/setup-python@v2
         with:
-          python-version: '3.6'
+          python-version: '3.9'
```

Comment:
Is there a reason for not going with 3.10?

Reply:
Not really. I'll bump to 3.10 and see what happens.